test/jdk/tools/jpackage/helpers/jdk/jpackage/test/LauncherAsServiceVerifier.java

```diff
@@ -225,7 +231,11 @@ static void verifyUninstalled(JPackageCommand cmd) {
     }
 
     static List<String> getLaunchersAsServices(JPackageCommand cmd) {
-        return Objects.requireNonNull(partitionLaunchers(cmd).get(true));
+        if (MacHelper.isForAppStore(cmd)) {
+            return List.of();
+        } else {
+            return Objects.requireNonNull(partitionLaunchers(cmd).get(true));
+        }
```

Comment:
Same as above, this is a redundant change: `partitionLaunchers()` function will return empty partition with the names of launchers installed as services after the tweak to the `boolean launcherAsService(JPackageCommand cmd, String launcherName)` function.